app/tests/test_app.py

```diff
@@ -0,0 +1,48 @@
+import unittest
+from app.app import app
+from app.config import API_VERSION
+
+
+class TestApp(unittest.TestCase):
+
+    def setUp(self):
+        app.testing = True
+        self.client = app.test_client()
+
+    def test_get_question(self):
+        response = self.client.get(API_VERSION + "/question/mcat/1?seed=123")
```

Comment:
I think I'd prefer having this version hardcoded here, so the tests will pick up any breaking changes to the existing API if we changed that param